content-src/components/A11yLinkButton/A11yLinkButton.jsx

```diff
@@ -0,0 +1,11 @@
+import {FormattedMessage} from "react-intl";
+import React from "react";
+
+export function A11yLinkButton(props) {
+  return (
+    <button {...props} className="a11y-link-button">
+      {props.children}
+      <FormattedMessage id="topsites_form_use_image_link" />
```

Comment:
Since this is specific to the top sites form, in this case you actually want this to be in `TopSiteForm.jsx`, i.e.
```jsx
<A11yLinkButton onClick={this.onEnableScreenshotUrlForm} className="enable-custom-image-input">
  <FormattedMessage id="topsites_form_use_image_link" />
</A11yLinkButton>
```